model/schema/core.yaml

```diff
@@ -156,28 +202,40 @@ classes:
           - value: gene_function_summary
           - value: gene_phenotype_summary
 
+  NoteDTO:
+    is_a: AuditedObjectDTO
+    slots:
+      - free_text
+      - note_type_name
+      - evidence_curies
+    slot_usage:
+      free_text:
+        required: true
+      note_type_name:
+        required_true
+
   SlotAnnotation:
```

Comment:
remove this class?

Reply:
Yeah, I think there's probably a bunch of cleanup to be done before moving this PR from draft - just wanted to get feedback first